WordPress/WordPressTest/WPAccount+LookupTests.swift

```diff
@@ -89,10 +89,15 @@ class WPAccountLookupTests: CoreDataTestCase {
     }
 
     func testLookupNumberOfAccountsReturnsCorrectValue() throws {
-        let _ = AccountBuilder(contextManager).build()
-        let _ = AccountBuilder(contextManager).build()
-        let _ = AccountBuilder(contextManager).build()
+        makeAccount()
+        makeAccount()
+        makeAccount()
 
         try XCTAssertEqual(WPAccount.lookupNumberOfAccounts(in: contextManager.mainContext), 3)
     }
+
+    @discardableResult
+    func makeAccount(_ additionalSetup: (AccountBuilder) -> (AccountBuilder) = { $0 }) -> WPAccount {
+        additionalSetup(AccountBuilder(contextManager.mainContext)).build()
```

Comment:
`{ $0 }` is an identity function for `(AccountBuilder) -> AccountBuilder`. It returns its input untouched, and is the perfect default parameter for what we want to do in this method.

```swift
// 1. get an account builder
let builder = AccountBuilder(mainContext)
// 2. configure the account it should build
  .with(something: ...)
  .with(somethingElse: ...)

// 3. build the account
return builder.build()
```

The code inlines the `AccountBuilder` instantiation within the setup closure, and calls `build` on the closure's return value, which is the configured builder.